api-report/driver-definitions.api.md

```diff
@@ -172,6 +172,7 @@ export interface IDocumentStorageService extends Partial<IDisposable> {
     createBlob(file: ArrayBufferLike): Promise<ICreateBlobResponse>;
     downloadSummary(handle: ISummaryHandle): Promise<ISummaryTree>;
     getSnapshotTree(version?: IVersion, scenarioName?: string): Promise<ISnapshotTree | null>;
+    getTime?(): number;
```

Comment:
i don't think we should be modifing the driver contract. this is a major change that we'd need to align with all of our server partners so they could add apis to their services to support. this seems beyond the scope of this work